migrating/ossm-migrating-premigration-checklist-assembly.adoc

```diff
@@ -0,0 +1,55 @@
+// Module included in the following assemblies:
+//
+// * service-mesh-docs-main/about/ossm-migrating-read-me-assembly.adoc
+
+:_mod-docs-content-type: ASSEMBLY
+[id=ossm-migrating-premigration-checklist-assembly]
+= Migrating from 2.6.z to 3.0 premigration checklist
+include::_attributes/common-attributes.adoc[]
+:context: ossm-migrating-migration-guides-assembly
+
+toc::[]
+
+Before you start migrating your `controlplane` from {SMProductName} 2.6 to 3.0, do the following:
+
+* [ ] Read "Important information to know if you are migrating from OpenShift Service Mesh 2.6".
+* [ ] Upgrade the {SMProduct} Operator to the latest 2.6.z release.
+* [ ] Upgrade your `ServiceMeshControlPlane` version to the latest {SMProduct} 2.6.z release.
+
+== Disable addons on your `ServiceMeshControlPlane`
+
+These addons are not supported in {SMProduct} 3.0 and must be disabled before migrating.
+
+** [ ] Disable network policy management by setting `spec.security.manageNetworkPolicy=false`. If you wish to keep the Network Policies created by the 2.6 ServiceMeshControlPlane, you will need to recreate and manage these manually.
+** [ ] Disable Prometheus by setting `spec.addons.prometheus.enabled=false`. See "User workload monitoring" for instructions configuring {SMProduct}h 3.0 with OpenShift Monitoring as a replacement for Prometheus.
```

Comment:
Users may want to make this change before migrating to OSSM 3, in which case, we should link to the 2.x documentation for using user-workload monitoring, located here: https://docs.openshift.com/container-platform/4.17/service_mesh/v2x/ossm-observability.html#ossm-integrating-with-user-workload-monitoring_observability.